doc/changes/devel/12877.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Fix bug where creating a :class:`~mne.time_frequency.SpectrumArray` from a
+:class:`numpy.ndarray` would require online the 'good' data channels to be present in
```

Comment:
```suggestion
:class:`numpy.ndarray` would require only the 'good' data channels to be present in
```

is this the correct intent?

Reply:
Typo and let's reformulate, thanks.

Reply:
See if the new changelog entry looks better to you.